src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,136 @@
+import { applyFilter } from 'src/api';
+import { filters } from './data-mocks/filters';
+
+const makePossibleFeature = (value) => [
+  {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0]
+    },
+    properties: {
+      column1: value
+    }
+  },
+  {
+    column1: value
+  }
+];
+
+describe('Filters', () => {
+  test('should return 1 if no filters present', () => {
+    const params = { filters: {}, type: 'number' };
+    expect(applyFilter(params)()).toBe(1);
+  });
+
+  test('should return true if no filters present', () => {
+    const params = { filters: {}, type: 'boolean' };
+    expect(applyFilter(params)()).toBe(true);
+  });
+
+  describe('feature passes filter - boolean type', () => {
+    const params = { filters, type: 'boolean' };
+
+    describe('should return false if feature column value is falsy', () => {
+      const testCases = [0, null, undefined, false, ''];
```

Comment:
if you apply my previous suggestions, this would be 'columnValues' instead of testCases..